custom/icds_reports/management/commands/nos_of_deaths.py

```diff
@@ -31,28 +30,36 @@ def handle(self, file_path, *args, **options):
         adapter = get_indicator_adapter(config)
         session_helper = connection_manager.get_session_helper(adapter.engine_id)
         person_table_name = get_table_name(domain, PERSON_TABLE_ID)
-        awc_location_table_name = get_table_name(domain, AWC_LOCATION_TABLE_ID)
         session = session_helper.Session
-
+        buckets = [('0 months', '6 months'),
+                   ('6 months', '2 years'),
+                   ('2 years', '6 years'),
+                   ('6 years', '14 years'),
+                   ('14 years', '18 years')
+                   ]
+        # 200 because we need 18+  and person of age more than 200 is not expected
+        buckets.append(('18 years', '200 years'))
```

Comment:
why is this appended (and not in the list above)

Reply:
Just for clearity what it actually means. Explained in inline comment

Reply:
I think you can put an inline comment in the list defininition like
```
[
1,
2,
# comment
3
]
```